lab_1_keywords_tfidf/main.py

```diff
@@ -99,12 +139,19 @@ def calculate_tfidf(term_freq: dict[str, float], idf: dict[str, float]) -> Optio
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(term_freq, dict) and isinstance(idf, dict):
+        tfidf_dic = {}
+        for key in term_freq.keys():
+            if idf[key] != 0:
+                tfidf_dic[key] = term_freq[key]*idf[key]
+            else:
+                tfidf_dic[key] = term_freq[key]*47
```

Comment:
IDF formula is as follows: 

$$idf(t, D) = log(\frac{|D|}{|\{d_i \in D : t \in d_i\}| + 1})$$